modules/temp-deck/temp-deck-arduino/lights.cpp

```diff
@@ -168,7 +168,17 @@ void Lights::startup_animation(int target_number, int transition_time) {
   set_color_bar(0, 0, 0, 1);
 }
 
-void Lights::setup_lights() {
+void Lights::setup_lights(bool is_blue_pin_5) {
```

Comment:
We can't add the logic to ctor since we won't know whether `blue_is_pin_5` or not until we check the tempdeck's version # in setup().
I'll assign them default values in the ctor and then update them in setup()